README.md

```diff
@@ -1,4 +1,48 @@
-HtUserRegistrationDoctrineORM
-=============================
-
-Doctrine2 ORM storage adapter for HtUserRegistration.
+HtUserRegistrationDoctrineORM
+=============================
+
+This module extends [HtUserRegistration](https://github.com/hrevert/HtUserRegistration)
+by replacing the Zend DB code with Doctrine.
+
+Install
+-------
+
+Require with composer
+```
+composer require hrevert/ht-user-registration-doctrine-orm
+```
+
+Include in config/application.config.php
+```php
+    'modules' => array(
+        ...
+        'HtUserRegistration',
+        'HtUserRegistrationDoctrineORM',  // include after HtUserRegistration
+        ...
+    ),
+```
+
+
+Configure
+----------
+
+Your User entity must implement `HtUserRegistrationDoctrineORM\Entity\UserInterface`.

```

Comment:
This interface guides the developer to implement many to one relations so the database can be browsed like `$user->getUserRegistrationRecord();`